mobile/app/src/main/res/values/strings.xml

```diff
@@ -113,6 +113,8 @@
         Note: IITCm can still be controlled via the Navigation Drawers</string>
     <string name="pref_force_desktop">Force desktop mode</string>
     <string name="pref_force_desktop_sum">Nice for tablets, looks awful on smartphones</string>
+    <string name="pref_dex_desktop">Force Samsung DeX desktop mode</string>
```

Comment:
To nit-pick:
Technically DeX is supported whether this is selected or not, this setting just forces IITC to always open in Desktop mode when running on an Active DeX system rather than having to toggle Desktop on and off when switching between and Active DeX system and normal mobile phone mode. 

But, this would work.

Reply:
I can easily make the change in my copy and submit it. What is the thought of making it on/off by default?  I really thing that anyone using a DeX mode - which is the only time this would be of any use would want it on.